pkgs/development/node-packages/main-programs.nix

```diff
@@ -36,7 +36,6 @@
   less = "lessc";
   localtunnel = "lt";
   lua-fmt = "luafmt";
-  neovim = "neovim-node-host";
```

Comment:
You forgot to remove it from lockfile, no?

Reply:
https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/node-packages/node-packages.json

Reply:
Gotcha, there was indeed still a neovim entry in there. Thanks for catching that. Removed.